setup.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python3
+
+from setuptools import setup
+
+with open("README.md", 'r') as f:
+    long_description = f.read()
+
+with open("requirements.txt", 'r') as f:
+    requirements = [i.strip() for i in f.readlines()]
+
+setup(
+    name="xCell",
+    version="dev",
+    description=("Pipeline to compute the Nx2pt angular power spectra and their covariances."),
```

Comment:
can you remove "the" here? Merge after that!

Reply:
Done. Merging! Thanks!